scripts/tests/test_safe_template_linter.py

```diff
@@ -730,3 +737,53 @@ def test_check_underscore_no_escape_allowed(self, data):
         linter._check_underscore_file_is_safe(data['template'], results)
 
         self.assertEqual(len(results.violations), 0)
+
+
+@ddt
+class TestJavaScriptLinter(TestLinter):
+    """
+    Test JavaScriptLinter
+    """
+
+    @data(
+        {'template': 'var m = "Text " + message + " text."', 'rule': None},
+        {'template': 'var m = "<p>" + message + "</p>"', 'rule': Rules.javascript_concat_html},
+        {'template': 'var m = " <p> " + message + " </p> "', 'rule': Rules.javascript_concat_html},
+    )
+    def test_concat_with_html(self, data):
+        """
+        Test _check_javascript_file_is_safe with concatenating strings and HTML
+        """
+        linter = JavaScriptLinter()
+        results = FileResults('')
+
+        linter._check_javascript_file_is_safe(data['template'], results)
+
+        self._validate_data_rule(data, results)
+
+    @data(
+        {'template': 'test.append( test.render().el )', 'rule': None},
+        {'template': 'test.append( test.render().$el )', 'rule': None},
+        {'template': 'test.append( testEl )', 'rule': None},
+        {'template': 'test.append( testEl )', 'rule': None},
+        {'template': 'test.append( $test )', 'rule': None},
```

Comment:
I currently have a rule to accept single variables with names that match the convention `anythingEl` or `$anything`. Does this make sense to you?